pkgs/development/lua-modules/nfd/default.nix

```diff
@@ -22,15 +21,22 @@ buildLuarocksPackage {
   ];
   knownRockspec = "lua/nfd-scm-1.rockspec";
 
-  luarocksConfig.LUA_LIBDIR = "${lua}/lib";
+  luarocksConfig.variables.LUA_LIBDIR = "${lua}/lib";
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = lib.optionals stdenv.isDarwin [ AppKit ];
 
-  fixupPhase = ''
+  postInstall = ''
     find $out -name nfd_zenity.so -execdir mv {} nfd.so \;
   '';
 
+  doInstallCheck = true;
+  installCheckInputs = [ lua51Packages.busted ];
```

Comment:
use `lua.pkgs.busted` instead since this will mix a 5.1 version when installing lua_52.pkkgs.nfd